### PR TITLE
Renaming MultiSelectTypeahead to ChipTypeahead, ux improvements (#1381)

### LINKED ISSUE #1381: Improve functionality of MultiSelectTypeahead
- [ ]  Instead of toggling, an item should be disabled if its already selected
- [ ]  Add a property that would allow an item to be selected more than once (`duplicates`)
- [ ]  Rename to `ChipTypeahead`

### PR DESCRIPTION
**Pull request by rorticus**
**Type:** bug / feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [x] Unit tests are included in the PR
* [ ] For new widgets, an entry has been added to the `.dojorc`
* [ ] For new widgets, `theme.variant()` is added to the root domnode
* [ ] Any widget variant uses `theme.compose` like [this](https://github.com/dojo/widgets/issues/847)
* [ ] WidgetProperties are exported

**Description:**

- Renaming `MultiSelectTypeahead` to `ChipTypeahead`
- Disabling `ChipTypeahead` options when they are selected (removing "toggle" feature)
- Adding a `duplicates` option that will allow the user to select more of the same value

Resolves #1381

### CODE REVIEW

File: src/chip-typeahead/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# @dojo/widgets/chip-typeahead
+
+Dojo's `ChipTypeahead` widget allows users to select multiple values from a dropdown of preconfigured options. This widget composes the the `Typeahead` and `Chip` widgets.
```
Comment (ericos) [inline src/chip-typeahead/README.md:3]:
```suggestion
Dojo's `ChipTypeahead` widget allows users to select multiple values from a dropdown of preconfigured options. This widget composes the `Typeahead` and `Chip` widgets.
```

File: src/theme/dojo/index.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ import * as listItem from './list-item.m.css';
 import * as loadingIndicator from './loading-indicator.m.css';
 import * as list from './list.m.css';
 import * as menuItem from './menu-item.m.css';
-import * as multiSelectTypeahead from './multi-select-typeahead.m.css';
+import * as multiSelectTypeahead from './chip-typeahead.m.css';
```
Comment (ericos) [inline src/theme/dojo/index.ts:29]: Import this as `chipTypeahead` for consistency?

File: src/theme/material/index.ts [hunk 3]
```diff
@@ -26,7 +26,7 @@ import * as listItem from './list-item.m.css';
 import * as loadingIndicator from './loading-indicator.m.css';
 import * as list from './list.m.css';
 import * as menuItem from './menu-item.m.css';
-import * as multiSelectTypeahead from './multi-select-typeahead.m.css';
+import * as multiSelectTypeahead from './chip-typeahead.m.css';
```
Comment (ericos) [inline src/theme/material/index.ts:29]: `chipTypeahead`?

File: src/theme/dojo/index.ts (hunk 2 above)
Comment (rorticus) [reply src/theme/dojo/index.ts:29]: Good catch!

**Pull request merged**